designs/ble-mote.stanza

```diff
@@ -18,98 +18,92 @@ defpackage ocdb/designs/ble-mote :
   import ocdb/tolerance
 
 
-OPERATING-TEMPERATURE = [-20.0 70.0]
+OPERATING-TEMPERATURE = [-20.0 50.0]
 OPTIMIZE-FOR = ["cost"]
 MIN-PKG = "0402"
+MIN-CAP-VOLTAGE = 12.0
 
-val board-shape = Rectangle(30.0, 18.0)
 
 ; PMS7003 particle counter interface via header pins
-pcb-module pms7003 :
+pcb-bundle pms7003-interface :
   port vin : power
   port vin-mcu : power
   pin reset
   pin set
   port uart : uart()
+
+pcb-module pms7003 :
+  port interface : pms7003-interface
+  
```

Comment:
Interesting -- create an interface to a module as a bundle which makes it more efficient and concise to access. I like this approach.

Reply:
Yes! It works well for systems comprised by multiple boards as well - the bundles stack together really nicely